weather-booking-app-client/src/components/ConfirmBookingDetails/ConfirmBookingDetails.tsx

```diff
@@ -61,18 +61,21 @@ const ConfirmBookingDetails: React.FC<ConfirmBookingDetailsProps> = (props) => {
       <IonCard className="confirm-booking-details-card">
         <IonCardContent className="booking-detail-container">
           <div className="booking-details-content">
-            <div className="booking-details-details">
-              <IonCardTitle className="booking-details-details__title">
+            <div className="booking-details-details" style={{ paddingTop: 0 }}>
+              <IonCardTitle className="booking-details-details__title" style={{ fontSize: "1rem" }}>
```

Comment:
Can we put this inside a css file?

Reply:
honestly just leave, our css is completely scuffed because they its all global. I made it inline style so because theres other elements that share same className and I don't to effect those ones.